### PR TITLE
updated few files

### PR DESCRIPTION
**Pull request by madhusimhadri**

### CODE REVIEW

File: src/app/weather/weather.component.html [hunk 1]
```diff
@@ -2,6 +2,8 @@ <h2 class="title">
   <span style="background-color: yellowgreen; padding: 15px; border-radius: 2px"
     >Weather App</span
   >
+
+  <h1>Practicing Git pull request on this repository</h1>
 </h2>
```
Comment (wake8) [inline src/app/weather/weather.component.html:8]: simulation comment, muskoni malli commit chey

File: src/app/weather/weather.component.ts [hunk 2]
```diff
@@ -60,4 +60,7 @@ export class WeatherComponent {
       this.getWeather();
     }
   }
+
+  //Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda, earum. Dicta dolores molestias
+
```
Comment (wake8) [inline src/app/weather/weather.component.ts:65]: simulation comment, more work needed

**Review by wake8 — CHANGES_REQUESTED** [review]
added simulation comments

File: src/app/weather/weather.component.ts (hunk 2 above)
Comment (madhusimhadri) [reply src/app/weather/weather.component.ts:65]: updated the changes

File: src/app/weather/weather.component.html (hunk 1 above)
Comment (madhusimhadri) [reply src/app/weather/weather.component.html:8]: changed

**Review by wake8 — APPROVED** [review]
LFTM

**Pull request merged**